best-practices/pod-controls-limits.yaml

```diff
@@ -0,0 +1,157 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: limit-pod-resorces
+  annotations:
+    policies.kyverno.io/title: Limited pod resources
+    policies.kyverno.io/category: K8s
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      Control and limited, limits and requests limit.
```

Comment:
Since this is composed of multiple rules, can you please explain in the description what all these do?